test/e2e/multihoming.go

```diff
@@ -244,7 +312,9 @@ var _ = Describe("Multi Homing", func() {
 						By("asserting the server pod has an IP from the configured range")
 						Expect(netStatus[0].IPs).NotTo(BeEmpty())
 						serverIP = netStatus[0].IPs[i]
-						Expect(inRange(popCIDRsPerNodePrefix(subnet, netPrefixLengthPerNode), serverIP)).To(Succeed())
+						cidrRange, err := getCIDRRange(subnet)
```

Comment:
why are you making this change  ? It works without it, doesn't it ?

Reply:
My thought is that get cidrRange based on form of cidr is better than assuming that the nodeSubnet is 24. no strong opinion though, I can change it back if you prefer.

Reply:
nothing is being assumed, we're using a constant whose name documents what is happening. 

You can even update the code to pass the attribute via networkConfig in each test if you prefer (and only set it for layer3).

IMO, that's preferable.

Reply:
ok, reverted.